insurance-engine/src/insurance_claims.rs

```diff
@@ -27,10 +29,21 @@ pub struct InsuranceClaimCollection {
     pub claims: Vec<InsuranceClaim>,
 }
 
-pub fn stake_authorities_filter(whitelist: HashSet<String>) -> Box<dyn Fn(&StakeMeta) -> bool> {
+pub fn stake_authorities_filter(whitelist: HashSet<Pubkey>) -> Box<dyn Fn(&StakeMeta) -> bool> {
     Box::new(move |s| whitelist.contains(&s.stake_authority))
 }
 
+pub fn stake_authorities_filter_string(
```

Comment:
Why do we need this with _string? we can just pass Pubkeys in (parse Pubkeys in the argument of the cli directly)

Reply:
Ok, I haven't realized. I was thinking that I need some manual conversion to Pubkey but it's about implementing the `FromStr` trait. PR updated, https://github.com/marinade-finance/validator-bonds/pull/19/commits/07b6236332d66ee7fada4cdfe9c0c37c47a62f81